actionview/test/actionpack/controller/render_test.rb

```diff
@@ -1427,7 +1427,7 @@ def test_partial_collection_shorthand_with_different_types_of_records
 
   def test_empty_partial_collection
     get :empty_partial_collection
-    assert_equal " ", @response.body
+    assert_equal "", @response.body
```

Comment:
We should still support rendering empty partial collections, but `""` feels like a more appropriate thing to render than `" "`.